Readme.md

```diff
@@ -288,6 +293,23 @@ Object.defineProperty(location, 'title', {
 })
 ```
 
+### Verify mock calling 
```

Comment:
I think we have enough examples in the “Matchers” section. It probably could be improved, but adding a new sections that is almost the same isn't a good solution.